protected/humhub/modules/space/views/membership/requestMembership.php

```diff
@@ -34,22 +38,26 @@
         </div>
         <div class="modal-footer">
             <hr/>
-            <?= AjaxButton::widget([
-                'label' => Yii::t('SpaceModule.base', 'Send'),
-                'ajaxOptions' => [
-                    'type' => 'POST',
-                    'beforeSend' => new yii\web\JsExpression('humhub.require("ui.modal").footerLoader'),
-                    'success' => new yii\web\JsExpression('humhub.require("ui.modal").setContent'),
-                    'url' => $space->createUrl('/space/membership/request-membership-form'),
-                ],
-                'htmlOptions' => [
-                    'class' => 'btn btn-primary'
+            <?= Html::a(
+                Yii::t('SpaceModule.base', 'Send'), '#', [
+                    'class' => ['btn', 'btn-primary'],
+                    'data' => [
+                        'action-click' => 'space.requestMembershipSend',
+                        'action-url' => $space->createUrl('/space/membership/request-membership-form'),
+                    ]
+                ]
+            ) ?>
+
+            <?= Html::button(
+                Yii::t('SpaceModule.base', 'Close'), [
+                    'class' => ['btn', 'btn-primary'],
+                    'data' => [
+                        'dismiss' => 'modal',
+                    ],
                 ]
-            ]); ?>
+            ) ?>
 
-            <button type="button" class="btn btn-default" data-dismiss="modal">
-                <?= Yii::t('SpaceModule.base', 'Close'); ?>
-            </button>
+            <?= LoaderWidget::widget(['id' => 'send-loader', 'cssClass' => 'loader-modal hidden']); ?>
```

Comment:
Do we really manually need to include the loader here. Is there no more generic approach?

Maybe you can find some generic solution here: 
https://github.com/humhub/humhub/pull/6389

Reply:
okay